fplus-lib/src/core/application/file.rs

```diff
@@ -48,6 +48,11 @@ pub struct ApplicationFile {
     pub id: String,
     #[serde(rename = "Issue Number")]
     pub issue_number: String,
+    #[serde(
+        rename = "Issue Reporter Handle",
+        skip_serializing_if = "Option::is_none"
+    )]
+    pub issue_reporter_handle: Option<String>,
```

Comment:
Decide if we put this field as a separate column or in the app file. We can't have it in both places, as it risks desyncs.